02-express-tutorial/app.js

```diff
@@ -1,12 +1,30 @@
 console.log('Express Tutorial');
 const express = require('express');
-const {products} = require('./data');
+const { products, people } = require('./data');
```

Comment:
The `people` import is not being used in this file since you extracted the logic that uses it to a controller. You can remove it.